setup.md

```diff
@@ -179,10 +180,19 @@ It is also important to make sure the limits are high enough:
 ip mptcp limits set [ subflows NR ] [ add_addr_accepted NR ]
 ```
 
-`subflows` is the limit of created and accepted subflows (paths), and
-`add_addr_accepted` is the limit of accepted `ADD_ADDR` -- IP address
-notification from the other peer -- that will result in the creation of
-subflows.
+- `subflows` is the limit of additional created and accepted subflows (paths),
+  for both the client and server sides (default is 2).
+- `add_addr_accepted` is the limit of accepted `ADD_ADDR` -- IP address
+  notification from the other peer -- that will result in the creation of
+  subflows, typically only for the client side (default is 0).
+
+{: .note}
+It is possible to reach the limits with fewer established subflows than
+expected, e.g. when new subflow requests cannot reach the other peer. In case of
+problem, please increase the limits, use `ss -Mai` to check the counters, and
+set the adapted routing or firewall rules to avoid using certain paths between
+specific IP addresses, e.g. in a lab setup with dedicated links, don't use
+default but specific routes.
```

Comment:
```suggestion
specific IP addresses. For example, in a lab setup with dedicated links use
specific routes rather than letting the kernel select the default route.
```